mqtt/fps_extracter.py

```diff
@@ -0,0 +1,80 @@
+import re
```

Comment:
this `mqtt` folder should be moved under source and renamed `performance-dashboard` and moved under `src`